src/pages/contributors/valentin-delaye.adoc

```diff
@@ -0,0 +1,55 @@
+= Valentin Delaye
+:page-name: Valentin Delaye
+:page-linkedin: 
+:page-twitter: 
+:page-github: jonesbusy
+:page-email: 
+:page-image: avatar/valentin-delaye.png
+:page-pronouns: He/Him/His
+:page-location: Lausanne, Switzerland
+:page-firstcommit: 2018
+:page-datepublished: 2024-02-07
+:page-featured: true
+:page-intro: Valentin is a Jenkins contributor and plugin maintainer passionate about technology and programming. This passion is so great that it has spread as his profession and hobby. He believes strongly in the power of open source and has been a Linux user since his teenage years. Valentin believes that open source is about consuming, taking advantage of, and contributing to it.
+
+== What is your background prior to contributing to Jenkins?
+
+Jenkins is the first open-source project I contributed to with a community.
+Before that, I had submitted some small contributions on homemade Maven plugins.
+
+Now, I'm contributing to different tools and technologies like Helm charts (link:https://github.com/bitnami/charts/tree/main/bitnami/external-dns[Bitnami external DNS]) and Ansible collections (link:https://github.com/prometheus-community/ansible[Prometheus collection]).
+Recently I've added some features to link:https://github.com/openrewrite/rewrite[OpenRewrite], which is also used by the Jenkins project.
+
+== How long have you been using Jenkins?
+
+I've been using Jenkins for quite a long time.
+I think the first time I used Jenkins was around 2009 before it was even called Jenkins.
+My first contribution to Jenkins occurred in 2018.
+
+== Why choose Jenkins over other projects?
+
+I gravitated towards Jenkins for its cost, flexibility, and ability to circumvent on-premises infrastructure constraints.
+Since Jenkins is an open-source project, it can be easily extended via plugins.
+
+== What problems has Jenkins solved for you?
+
+Jenkins has helped me with automation and employing continuous integration despite on-premises environment limitations.
+
+== Is there an aspect of Jenkins that you're particularly passionate about?
+
+The community of course!
+Jenkins also has a massive plugin ecosystem that provides further customization and abilities.
+
+== What sort of contributions have felt the most successful or impactful?
+
+There isn't one contribution in particular, but over the last two years I've adopted about 30 previously abandoned plugins.
+I've refreshed them and fixed any issues, including security, and I now maintain them regularly.
```

Comment:
"Over the last two years I've adopted about 30 previously abandoned plugins. I've refreshed them and fixed any issues, including security, and I now maintain them regularly"

Can you add a new sentence (feel free to rephrase it)

"But, if I need to choose the contribution I'm the most proud of is the recent addition of OpenCover and NUnit parser into the coverage plugin"

Reply:
Of course, thanks very much @jonesbusy !